lib/js/jsfetch.nim

```diff
@@ -0,0 +1,139 @@
+## - Fetch for the JavaScript target: https://developer.mozilla.org/docs/Web/API/Fetch_API
```

Comment:
* move to `lib/std/jsfetch.nim` for reasons discussed in jsbigints PR
* ditto with for other file
* how about avoid such future regressions by (in another PR) adding some logic to CI so that CI fails if we attempt to add another file not under `lib/std` (essentially, by checking against a whitelist, similar to what's done in kochdocs)